client.py

```diff
@@ -0,0 +1,34 @@
+import socket
+
+ADDR = ('127.0.0.1', 8011) #port 0 may force os to find an open port
```

Comment:
port 0 would, but then you'd be hard-pressed to share that between the client and the server.